packages/visx-tooltip/src/hooks/useTooltipInPortal.tsx

```diff
@@ -50,21 +58,30 @@ export default function useTooltipInPortal({
         const zIndex = zIndexProp == null ? zIndexOption : zIndexProp;
         const TooltipComponent = detectBounds ? TooltipWithBounds : Tooltip;
         // convert container coordinates to page coordinates
-        const portalLeft = containerLeft + (containerBounds.left || 0) + window.scrollX;
-        const portalTop = containerTop + (containerBounds.top || 0) + window.scrollY;
+        const portalLeft = portalContainer
+          ? tooltipLeft - (portalContainerRect?.left || 0) + (containerBounds.left || 0)
```

Comment:
is `window.scrollX/Y` not relevant for these?

Reply:
They are not indeed. I don't know enough about the internals of React portals to explain exactly why, but in the case when a custom portal container is used, including `window.scrollX/Y` in the calculation of the tooltip position leads to bugs like this:

https://user-images.githubusercontent.com/6173255/201194569-76340e15-4a88-4d71-b0db-042068a06529.mov

Reply:
ah maybe when using a custom container, the window x/y is already accounted for. thanks for the video!